app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java

```diff
@@ -390,19 +390,21 @@ private void dismiss() {
 
     @Override
     public void onCommit() {
-        ViewUtils.hideKeyboard(urlView);
+        if (!urlView.getText().toString().isEmpty()) {
```

Comment:
nit: Let's avoid calling getText() and toString() multiple times by assigning it to some variable (final String input = ..)